examples/apps/echo/client/echo_client.py

```diff
@@ -201,12 +201,9 @@ def Main(args=None):
 	# Add worker input data
 	wo_params.add_in_data(message)
 
-	# Sign work order
-	private_key = utility.generate_signing_keys()
+	# Encrypt work order request hash
 	wo_params.add_encrypted_request_hash()
-	if wo_params.add_requester_signature(private_key) == False:
```

Comment:
Why do we need to remove signature part? Signature is optional doesn't mean to remove it everywhere. I think we should keep it. Since it is reference client and if some use this as reference then we should provide all option to add signature and verify signature.

Reply:
The issue with adding requesterSignature in request is we are also passing verifyingKey in JSON request to worker enclave which is outside the spec. So this may lead to confusion especially when 3rd parties are writing client applications in different language other than Python. My understanding is @manju956 will write a test case for requesterSignature as a reference code.

Reply:
The issue with adding requesterSignature in request is we are also passing verifyingKey in JSON request to worker enclave which is outside the spec. So this may lead to confusion especially when 3rd parties are writing client applications in different language other than Python. My understanding is @manju956 will write a test case for requesterSignature as a reference code.

Reply:
test case which includes requesterSignature and verifyingKey is already included in this PR.

Reply:
I have added command line option to enable requester signature